package.json

```diff
@@ -49,6 +49,7 @@
     "cssom": "^0.3.0",
     "domhandler": "^2.3.0",
     "escodegen": "^1.4.1",
+    "eslint": "^1.10.3",
```

Comment:
Hm since we use eslint in the devDependencies, do we want to keep it there or do we want to remove that?